bitcash/transaction.py

```diff
@@ -113,7 +113,10 @@ def sanitize_tx_data(unspents, outputs, fee, leftover, combine=True, message=Non
     messages = []
 
     if message:
-        message_chunks = chunk_data(message.encode('utf-8'), MESSAGE_LIMIT)
+        if not bytes:
```

Comment:
Isn't `bytes()` a bultin? Not sure if using bytes as a name will cause issues or just confusion.

Reply:
Ugh, Python taking all the good names...